### PR TITLE
Update condensable list with new stripped logs

### PR DESCRIPTION
**Pull request by Chew**
also changed the i.e. to something better, data values aren't really a thing anymore.

### CODE REVIEW

File: condensables.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Condensable items
 
 # Condensable items
 
-The following list contains all the valid condensable items that can be used to generate Artron Energy. The number represents the relative energy value (i.e. Item value:1 != Energy value:1).
+The following list contains all the valid condensable items that can be used to generate Artron Energy. The number represents the relative energy value, for example, 1 Apple will give you 10 Artron Energy.
```
Comment (eccentricdevotion) [inline condensables.md:10]: Actually an Apple will only give you [5 Artron energy](https://github.com/eccentricdevotion/TARDIS/blob/v4.0/src/main/java/me/eccentric_nz/TARDIS/artron/TARDISCondenserListener.java#L202)

Comment (Chew) [reply condensables.md:10]: Really? Why does the actual list say 10? I just checked and yeah it's 5

Comment (Chew) [reply condensables.md:10]: Oh I get it. The relative value is twice the Artron energy given? Where is the relative value used?

Comment (eccentricdevotion) [reply condensables.md:10]: It's never used, I'm just too lazy to change all the current values to half what they are now

Comment (Chew) [reply condensables.md:10]: I see a PR incoming...

Comment (Chew) [reply condensables.md:10]: Hm. Wouldn't that break existing configs that account for the x2?

Comment (eccentricdevotion) [reply condensables.md:10]: Go for it, but as far as code optimisation goes there'd be only an infinitesimal benefit.

Comment (eccentricdevotion) [reply condensables.md:10]: Only the values from the condensables list are halved all other values are 1:1 eg artron.yml

**Pull request closed without merging**